base/strings/string.jl

```diff
@@ -190,14 +191,71 @@ end
 end
 
 ## checking UTF-8 & ACSII validity ##
+# This table is used by the shift base DFA validation for UTF-8
+const _UTF8_DFA_TABLE = [
+    fill(UInt64(109802048057794944), 128);
+    fill(UInt64(113232530780455302), 16);
+    fill(UInt64(109855655693648262), 16);
+    fill(UInt64(109855649351860614), 32);
+    fill(UInt64(109802048057794950), 2);
+    fill(UInt64(109802048057794956), 30);
+    fill(UInt64(109802048057794968), 1);
+    fill(UInt64(109802048057794962), 12);
+    fill(UInt64(109802048057794974), 1);
+    fill(UInt64(109802048057794962), 2);
+    fill(UInt64(109802048057794980), 1);
+    fill(UInt64(109802048057794986), 3);
+    fill(UInt64(109802048057794992), 1);
+    fill(UInt64(109802048057794950), 11)
+    ]::Vector{UInt64}
```

Comment:
the type assert here doesn't do anything.

Reply:
Removed